backend/curation/api/curation-api.yml

```diff
@@ -14,6 +14,27 @@ info:
     ## Data Model
     <!-- Data model diagram created via Lucid: https://lucid.app/lucidchart/8d085832-34f4-4413-9c1d-44e77d06e326/edit?invitationId=inv_ed354532-7256-4e0a-9ad0-b622905e5198&page=Vfzkzdv~I1fL# -->
     <img src="/static/discover-api-data-model.png" width="50%"/>
+
+    ## Working with versions
+
+    #### Access to prior published version history
+    The CZ CELLxGENE Discover API employs the concept of *Versions* for our two core entities, *Collections* and
+    *Datasets*. This is to permit access to prior published instances (versions) of these entities even as new updates
+    for them are published.
+
```

Comment:
The CZ CELLxGENE Discover API supports access to all published versions of Collections and Datasets. For example, if a Dataset was published and then later updated and published again, then both Dataset Versions can be accessed.

Reply:
General comment. It's help to have line breaks for reviewers of markdown. Otherwise, it's hard to place the review comment.

Reply:
You mean one line per sentence...? I'm not quite sure what you mean.

Reply:
Maximum number of characters per line ... 

See https://about.gitlab.com/blog/2016/10/11/wrapping-text/ or https://peps.python.org/pep-0008/#maximum-line-length ...

Reply:
Basically, I should be able to comment line by line in a PR.

Reply:
You can comment line by line. I was just following our PyCharm guide for line length, which is longer than is suggested by the links you put above